Tests/SQSQueueImplementationTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Wowo\QueueBundle\Tests;
+
+use \Guzzle\Service\Resource\Model;
+use \Wowo\QueueBundle\Implementation\SQSQueueImplementation;
+
+class SQSQueueImplementationTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @test
+     * @dataProvider data_putBatch
+     */
+    public function putBatch($jobs, $batchCount)
+    {
+        $tube = 'tube';
+        $queueName = 'queueName';
+
+        $q = $this->getMockBuilder('\Aws\Sqs\SqsClient')->disableOriginalConstructor()->setMethods(array('sendMessage', 'sendMessageBatch', 'getQueueUrl'))->getMock();
```

Comment:
I tried that at first, but Mockery didn't like a final method in \Aws\Sqs\SqsClient, nor when I tried the partial mock with an instantiated class. I'll give it another look when I change the array notation.